code/drasil-example/projectile/lib/Drasil/Projectile/Lesson/Body.hs

```diff
@@ -54,11 +54,6 @@ symbMap = cdb (map dqdWr physicscon ++ symbols) (nw projectileMotion :
   map nw physicCon ++ concepts) 
   ([] :: [ConceptChunk]) ([] :: [UnitDefn]) [] [] [] [] [] [] allRefs []
 
-usedDB :: ChunkDB
-usedDB = cdb' ([] :: [DefinedQuantityDict]) (map nw symbols :: [IdeaDict]) ([] :: [ConceptChunk])
-  ([] :: [UnitDefn]) [] [] [] [] ([] :: [ConceptInstance])
-  ([] :: [LabelledContent]) ([] :: [Reference]) []
```

Comment:
Unused. Previously used for the table of abbreviations and acronyms (reworked in #4248).